registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/test/java/io/mosip/registration/processor/workflowmanager/service/test/UserNotificationApiTest.java

```diff
@@ -0,0 +1,522 @@
+            }
+
+            @Override
+            public boolean removeHeadersEndHandler(int handlerID) {
+                return false;
+            }
+
+
+
+            @Override
+            public boolean removeBodyEndHandler(int handlerID) {
+                return false;
+            }
+
+            @Override
+            public <T> T remove(String key) {
+                return null;
+            }
+
+            @Override
+            public MultiMap queryParams() {
+                return null;
+            }
+
+            @Override
+            public List<String> queryParam(String query) {
+                return null;
+            }
+
+            @Override
+            public RoutingContext put(String key, Object obj) {
+                return null;
+            }
+
+            @Override
+            public Map<String, String> pathParams() {
+                return null;
+            }
+
+            @Override
+            public String pathParam(String name) {
+                return null;
+            }
+
+            @Override
+            public ParsedHeaderValues parsedHeaders() {
+                return null;
+            }
+
+            @Override
+            public String normalisedPath() {
+                return null;
+            }
+
+            @Override
+            public io.vertx.ext.web.@Nullable Cookie getCookie(String s) {
+                return null;
+            }
+
+            @Override
+            public void next() {
+            }
+
+            @Override
+            public String mountPoint() {
+                return null;
+            }
+
+
+
+            @Override
+            public String getBodyAsString(String encoding) {
+                return null;
+            }
+
+            @Override
+            public String getBodyAsString() {
+                return null;
+            }
+
+//            @Override
+//            public JsonArray getBodyAsJsonArray() {
+//                return null;
+//            }
+
+            @Override
+            public JsonObject getBodyAsJson() {
+                JsonObject obj = new JsonObject();
+                obj.put("id", "mosip.registration.processor.message.sender");
+                obj.put("version", "1.0");
+                obj.put("requesttime", "2021-03-15T10:02:45.474Z");
+                JsonObject requestObject = new JsonObject();
+                requestObject.put("rid", "10003100030001520190422074511");
+                requestObject.put("regType", "NEW");
+                obj.put("request", requestObject);
+                return obj;
+            }
+
+            @Override
+            public @Nullable JsonArray getBodyAsJsonArray() {
+                return null;
+            }
+
+            @Override
+            public @Nullable Buffer getBody() {
+                return null;
+            }
+
+//            @Override
+//            public Buffer getBody() {
+//                return null;
+//            }
```

Comment:
If these kind of commented code is not required, please remove the same

Reply:
removed